bundler/config/Staging.toml

```
@@ -15,6 +15,7 @@ level = "info"
 
 [database]
 file = "toad.db"
+url = "postgres://<user>:<pwd>@<host>/<db_name>"
```

I think you can fill in everything except password here